modules/machineset-creating.adoc

```diff
@@ -198,17 +202,22 @@ $ oc get machineset -n openshift-machine-api
 .Example output
 [source,terminal]
 ----
+ifdef::win,localzone[]
+NAME                                       DESIRED   CURRENT   READY   AVAILABLE   AGE
 ifdef::win[]
-NAME                                      DESIRED   CURRENT   READY   AVAILABLE   AGE
-agl030519-vplxk-windows-worker-us-east-1a    1         1         1       1        11m
-agl030519-vplxk-worker-us-east-1a            1         1         1       1        55m
-agl030519-vplxk-worker-us-east-1b            1         1         1       1        55m
-agl030519-vplxk-worker-us-east-1c            1         1         1       1        55m
-agl030519-vplxk-worker-us-east-1d            0         0                          55m
-agl030519-vplxk-worker-us-east-1e            0         0                          55m
-agl030519-vplxk-worker-us-east-1f            0         0                          55m
+agl030519-vplxk-windows-worker-us-east-1a  1         1         1       1           11m
```

Comment:
Need to fix as it does not match https://github.com/openshift/openshift-docs/pull/63729/files